lk2nd/device/dts/msm8916/msm8939-huawei-rio.dts

```diff
@@ -18,7 +18,7 @@
 	lk2nd,dtb-files = "msm8939-huawei-rio";
 
 	panel {
-		compatible = "huawei,rio-panel";
+		compatible = "huawei,rio-panel", "lk2nd,panel";;
```

Comment:
Thanks for noticing this! Would you mind fixing the commit so it actually compiles on CI?

```suggestion
		compatible = "huawei,rio-panel", "lk2nd,panel";
```